### DISCUSSION TITLE
::ffff: in IP field

### DISCUSSION THREAD

**Discussion by knpshkn** [akvorado/akvorado#48] [Q&A] [answered]
Hi!

Is there a way to hide or remove `::ffff:` from the ip address fields?

**Comment by vincentbernat** [comment, marked as answer]
Currently not. However, it shouldn't be difficult to do that.

**Reply by vincentbernat** [reply]
This will be done by the next version: 085cb93e4ba0f9aa5f2dfa509938a3edf3b95092.

**Reply by knpshkn** [reply]
much appreciated!
